src/inatinqperf/benchmark/benchmark.py

```diff
@@ -294,3 +258,15 @@ def recall_at_k(approx_i: np.ndarray, exact_i: np.ndarray, k: int) -> float:
     for i in range(approx_i.shape[0]):
         hits += len(set(approx_i[i, :k]).intersection(set(exact_i[i, :k])))
     return hits / float(approx_i.shape[0] * k)
+
+
+def _ids_to_fixed_array(results: Sequence[SearchResult], topk: int) -> np.ndarray:
```

Comment:
We don't need this method. The responsibility of ensuring that we only get `k` results should be on the adaptor's `search` method.